tbdex/orderinstructions/orderinstructions.go

```diff
@@ -0,0 +1,258 @@
+package orderinstructions
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/TBD54566975/tbdex-go/tbdex/cancel"
+	"github.com/TBD54566975/tbdex-go/tbdex/closemsg"
+	"github.com/TBD54566975/tbdex-go/tbdex/crypto"
+	"github.com/TBD54566975/tbdex-go/tbdex/message"
+	"github.com/TBD54566975/tbdex-go/tbdex/validator"
+	"github.com/tbd54566975/web5-go/dids/did"
+	"go.jetpack.io/typeid"
+)
+
+// Kind is the value used within a message's metadata.kind
+const Kind = "orderinstructions"
+
+// ValidNext returns the valid message kinds that can follow an orderinstructions.
+func ValidNext() []string {
+	return []string{closemsg.Kind, cancel.Kind}
```

Comment:
an order can't follow an orderinstructions?

Reply:
or an orderstatus?

Reply:
order > order instructions > order status is the sequence. will update to add orderstatus to the list here. thanks!